ansible/roles/captain/vars/main.yml

```diff
@@ -1,5 +1,9 @@
 deploy_env: captain
 # supervisor_task_conf: "{{ service_home }}/supervisor"
+#
+# Defined in config directory:
+# softlayer_vpn_username: ...
+# softlayer_vpn_password: ...
```

Comment:
Sure just got backed so happy to talk about it this week or next. This has been deployed I can remove the creds until then. but just so i understand correctly your concern is having the credentials on hqcaptain not on salt machine right?